Remove hard-coded compiler path based on found Clang/LLVM, as this interferes
with our rpath wrapping

Author: Jan Andre Reuter (jan@zyten.de)

diff --color -Naur HIPIFY-rocm-6.4.1.orig/CMakeLists.txt HIPIFY-rocm-6.4.1/CMakeLists.txt
--- HIPIFY-rocm-6.4.1.orig/CMakeLists.txt	2025-04-15 23:59:06.000000000 +0200
+++ HIPIFY-rocm-6.4.1/CMakeLists.txt	2025-06-24 13:38:09.877030234 +0200
@@ -88,8 +88,10 @@
           ${LLVM_EXTERNAL_LLD_SOURCE_DIR}/include)
     endif()
   else()
-    set(CMAKE_CXX_COMPILER ${LLVM_TOOLS_BINARY_DIR}/clang++)
-    set(CMAKE_C_COMPILER ${LLVM_TOOLS_BINARY_DIR}/clang)
+    # With EasyBuild, we set these things externally, so no need to hard-code this.
+    # It also breaks building with rpath wrappers.
+    # set(CMAKE_CXX_COMPILER ${LLVM_TOOLS_BINARY_DIR}/clang++)
+    # set(CMAKE_C_COMPILER ${LLVM_TOOLS_BINARY_DIR}/clang)
   endif()
 
   # Link against LLVM and CLANG libraries.
